app/Resources/views/themes/tastefulplanet/full/form.html.twig

```diff
@@ -5,16 +5,14 @@
         <div class="container">
             <div class="row">
                 <div class="col-md-8 offset-md-2">
-                    <div class="field-description">
+                    <div class="block-header__name">
```

Comment:
I see you're trying to implement BEM naming convention. I'm missing the context of a `Block`. What you did here is adding a definition of `Element` from BEM.
This is incorrectly implemented code.